astrowidgets/__init__.py

```diff
@@ -17,7 +17,3 @@ class UnsupportedPythonError(Exception):
         pass
 
     raise UnsupportedPythonError("astrowidgets does not support Python < {}".format(__minimum_python_version__))
-
-if not _ASTROPY_SETUP_:
-    # For egg_info test builds to pass, put package imports here.
-    from .core import *
```

Comment:
You can still do `from .core import *`  here, but you don't need to do it inside the `if` block. 😅

Reply:
😮 🤦‍♂ Not sure how the tests passed with this...